ansible/roles/stack-sunbird/templates/search-service_application.conf

```diff
@@ -344,6 +344,6 @@ language.map={"Hindi":"hi", "English":"en", "Telugu":"te", "Kannada":"ka", "Tami
 channel.default="in.ekstep"
 compositesearch.index.name="{{ compositesearch_index_name }}"
 
-content.tagging.backward_enable=true
+content.tagging.backward_enable=false
```

Comment:
This need to be variablized and not hardcoded, will ask the team to do it in another PR.

Reply:
As this already gone libe, it needs to be done as a hotfix / in 3.1